lib/matplotlib/backends/backend_pdf.py

```diff
@@ -2358,7 +2367,9 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
             fonttype = 1
         else:
             font = self._get_font_ttf(prop)
-            self.file._character_tracker.track(font, s)
+            char_to_font = font.get_fontmap(s)
+            for _c, _f in char_to_font.items():
+                self.file._character_tracker.track(_f, _c)
```

Comment:
If this `for` loop happens everywhere, maybe it can go in `_character_tracker.track`?